docs/low-code-spark/spark-streaming/structured-streaming.md

```diff
@@ -0,0 +1,105 @@
+
+#### Formats Supported
+
+We provide support for the following formats at present. The Gem properties would be accessible under the Properties Tab, under a + sign:
+
+1. Json: Native Connector Docs for Source [here](https://spark.apache.org/docs/3.1.3/api/python/reference/api/pyspark.sql.streaming.DataStreamReader.json.html). Additional Autoloader Options [here](https://docs.databricks.com/ingestion/auto-loader/options.html#json-options)
+2. CSV: Native Connector Docs for Source [here](https://spark.apache.org/docs/3.1.3/api/python/reference/api/pyspark.sql.streaming.DataStreamReader.csv.html). Additional Autoloader Options [here](https://docs.databricks.com/ingestion/auto-loader/options.html#csv-options)
+3. Parquet: Native Connector Docs for Source [here](https://spark.apache.org/docs/3.1.3/api/python/reference/api/pyspark.sql.streaming.DataStreamReader.csv.html). Additional Autoloader Options [here](https://docs.databricks.com/ingestion/auto-loader/options.html#csv-options)
+4. Orc: Native Connector Docs for Source [here](https://spark.apache.org/docs/3.1.3/api/python/reference/api/pyspark.sql.streaming.DataStreamReader.orc.html). Additional Autoloader Options [here](https://docs.databricks.com/ingestion/auto-loader/options.html#orc-options)
+5. Delta: A quickstart on Delta Lake Stream Reading and Writing is available [here](https://docs.databricks.com/structured-streaming/delta-lake.html#delta-table-as-a-source). Connector Docs are available [here](https://docs.delta.io/latest/delta-streaming.html). Note, that this would require installing the Spark Delta Lake Connector if the user has an on prem deployment. We have additionally provided support for Merge in the Delta Lake Write Connector. (uses `forEatchBatch` behind the scenes)
+
+#### Warehouse Based Sources/Targets
+
+Warehouse-based sources and targets are also supported, with Splunk HEC currently being the sole example.
+
+#### Stream Based Sources/Targets
+
+Currently we support only **Kafka Steaming** Sources/Targets. A very comprehensive reading on what Kafka Sources/Targets supports is available [here](https://spark.apache.org/docs/latest/structured-streaming-kafka-integration.html).
+We currently allow a user to authenticate using username/password via Databricks Secrets. We DO NOT recommend using the username/password option directly as that would commit your credentials to your code.
+The Kafka Gem also has support for inferring the Schema of the incoming message (automatically the `value` column) if it is either JSON or AVRO type. A user would have to provide an example record for the same.
+
+### Lookup
+
+These are also same as Lookups applicable to Batch Pipelines. We can use lookups to interact with data coming into a Pipeline. More on Lookup [here](/low-code-spark/gems/source-target/lookup.md)
```

Comment:
"We can use lookups to interact with data coming into a Pipeline." -> "Lookups are a way to efficiently reuse small tables as part of Column expressions".